internal/util/k8s_test.go

```diff
@@ -45,3 +48,111 @@ func TestToClientObjects(t *testing.T) {
 	assert.Equal(t, &s[0], objs[0])
 	assert.Equal(t, &s[1], objs[1])
 }
+
+func Test_ObjNameFuncs(t *testing.T) {
```

Comment:
@jonstacks lmk your thoughts on these name formats. Personally I really like `<kind>/<name>` as a general pattern.

Reply:
I don't have a strong opinion on this yet. I'll probably wait to see which of them we use and where and how they impact debugging before having any thoughts on it.